problem_set_1.py

```diff
@@ -1,27 +1,40 @@
 # 1. Set the variable `given_name` to the string "Addison".
-
+given_name = "Addison"
 
 # ------------------------------------------------------------------------------
 # 2. You have 20 candies that you must divide equally among 6 people. How many candies will be left over?
 # Set variables for `candies`, `people`, `left_over` to make your tests pass.
-
+candies = 20 
+people = 6
+left_over = candies / people
 
 # ------------------------------------------------------------------------------
 # 3. Create a function called `greeting` that returns "Hello, <name>!",
 # where <name> is the name given as an argument to the function.
+def greeting():
+    print("hello <name>!")
```

Comment:
In this case, you want to use one of the Python functions that lets you combine strings and variables. This function has a parameter called `name` and will take a different argument for `name` every time it is called.